dashboard/src/utils/env.ts

```diff
@@ -7,17 +7,29 @@ export const LOCAL_MIGRATIONS_URL = `http://localhost:${
 }`;
 
 /**
- * Port of the locally running backend.s
+ * Port of the locally running backend.
  */
 export const LOCAL_BACKEND_PORT =
   process.env.NEXT_PUBLIC_NHOST_LOCAL_BACKEND_PORT;
 
+/**
+ * Subdomain of the Nhost project.
+ */
+export const SUBDOMAIN =
+  process.env.NEXT_PUBLIC_NHOST_LOCAL_SUBDOMAIN || 'localhost';
+
+/**
+ * Subdomain of the Nhost project with protocol.
+ */
+export const SUBDOMAIN_WITH_PROTOCOL =
+  SUBDOMAIN === 'localhost' ? SUBDOMAIN : `https://${SUBDOMAIN}`;
```

Comment:
it's confusing a bit, if it's subdomain with protocol, then maybe it should be `http://localhost` in case subdomain equals to `localhost`?